docs/documentation/exif.rst

```diff
@@ -0,0 +1,62 @@
+Exif
+====
+
+Vimiv provides optional exif support if either `pyexiv2`_ or `piexif`_ is available. If
+this is the case:
+
+#. Exif metadata is automatically copied from source to destination when writing images
+   to disk.
+#. The ``:metadata`` command and the corresponding ``i``-keybinding is available.
+#. The ``{exif-date-time}`` statusbar module is available.
+
+
+Advantages of the different exif libraries
+------------------------------------------
+
+`Pyexiv2`_ is the more powerful of the two options. One large advantage is that it has
+the option to format exif data into human readable format, for example ``FocalLength:
+5.0 mm`` where `piexif`_ would only give ``FocalLength: 5.0``. However, given it is
+written as python bindings to the c++ api of `exiv2`_, the installation is more involved
+compared to the pure python `piexif`_ module.
+
+We recommend to use `pyexiv2`_ if the installation is not too involved on your system
+and `piexif`_ as a fallback solution or in case you don't need the full power of
+`pyexiv2`_ and prefer something more lightweight.
+
+
+Moving from piexif to pyexiv2
+-----------------------------
+
+As pyexiv2 is the more powerful option compared to piexif, vimiv will prefer pyexiv2
+over piexif. Therefore, to switch to pyexiv2 simply install it on your system and vimiv
+will use it automatically.
+
+
+Finding valid metadata keys
```

Comment:
I would name this sections rather something like _Customize metadata keysets_ or _Customize metadata keysets and find valid metadata keys_, though this may be too long.